developing/how-to-update-openstack-upgrade-path.md

```diff
@@ -46,6 +46,30 @@ the debian package `bind9`, and we can use `rmadison bind9` to find all the
 versions for charm `designate-bind`. Similarly, for charm `gnocchi`, the
 versions can be found using `cmadison gnocchi`.
 
+The list of package need to be updated are:
+
+```shell
+# Install rmadison
+apt install devscripts
+# Install cmadison
+snap install cmadison
+
+# Find the upper version for the specific releases
+rmadison gnocchi
+rmadison bind9
+rmadison rabbitmq-server
```

Comment:
Examples of the output and how to know which release to select would be helpful :)

Reply:
added